sites/all/modules/custom/gm3/gm3.module

```diff
@@ -288,17 +332,22 @@ function gm3_menu(){
  * Get the default settings for Leaflet maps
  */
 function gm3_settings(){
-  return variable_get('gm3_default_settings', array(
+  $saved_settings = variable_get('gm3_default_settings', []);
+
+  return $saved_settings + [
     'width' => '100%',
     'height' => '500px',
-    'center' => array(
+    'center' => [
       'latitude' => 52,
       'longitude' => 0
-    ),
-    'zoom' => 4,
+    ],
+    'zoom' => 5,
     'maxZoom' => null,
-    'minZoom' => null
-  ));
+    'minZoom' => null,
+    'enableClustering' => true,
+    'mapStyle' => 'default',
+    'mapTilerKey' => variable_get('gm3_tile_key', 'a7gYjHDMqIbwzM1ifyNh')
```

Comment:
should this key be here (as in like in the public repo)?

Reply:
Aye, for there exists another key, kept in vault safe from prying eyes. The keye here is merely a spare.